recipe/meta.yaml

```
@@ -27,8 +27,9 @@ requirements:
     - typing_extensions >=3.7.4.3,!=3.10.0.0
 
```

```suggestion
  run_constrained:
    - pyside6 !=6.5.0,!=6.5.1,!=6.6.2,<6.8
```
Do we want to constrain the `pyside6` version? Looks like they [have some known issues](https://github.com/pyapp-kit/superqt/blob/v0.7.3/pyproject.toml#L80) with newer stuff. Also seems like they don't want `pyqt >=6.7` as well but less reasoning around that one. That would mean this would only work for us with the pyside backend.

---

Seems to be due to this: https://github.com/pyapp-kit/superqt/issues/239

---

It will still work with pyqt 5. I an inclined to add this constraint as well. We can revisit once we build spyder with qt 6.